monosi/anomalies/__init__.py

```diff
@@ -27,19 +36,30 @@ def _std_dev(cls, values: List[Any]):
         return std_dev
 
     @classmethod
-    def _z_scores(cls, stats: List[Any]):
-        values = [stat.value for stat in stats]
+    def _z_scores(cls, stats: List[MetricStat], filter_null=True):
+        stats_to_calc = stats
```

Comment:
Why bother with this line and rename?

Reply:
because filtering just the values isn't enough, we need to filter the stats list, otherwise the calculations at the bottom of the method for z score will not work

Reply:
I think it's redundant but won't block.